.eslintrc

```diff
@@ -28,7 +29,6 @@
     "quotes": [ 1, "single", "avoid-escape" ],
     "no-use-before-define": [ 2, { "functions": false } ],
     "semi": [1, "always"],
-    "prefer-const": 1,
```

Comment:
I think this is a good rule - any reason why it's gone?

Reply:
ah, yes, I think somewhere in the code (sorry forgot where 🤷‍♂️) it was changing a `let` it into a `const` and causing problems. But yeah, ok fair let's turn it back on, and when/if it resurfaces we can address it then and see what to do about it.